tools/proteore_go_terms_enrich_comparison/GO_terms_enrich_comparison.xml

```diff
@@ -0,0 +1,201 @@
+<tool id="go_terms_enrich_comparison" name="Biological theme comparison" version="2020.02.04">
+    <description>(Human, Mouse, Rat)[clusterProfiler]</description>
```

Comment:
Could gprofiler be used here as a replacement? https://github.com/galaxyproject/tools-iuc/tree/master/tools/gprofiler